climakitae/explore/warming.py

```diff
@@ -12,6 +12,12 @@
 from climakitae.core.paths import (
     gwl_1981_2010_file,
     gwl_1850_1900_file,
+    ssp119_file,
```

Comment:
@vicford These ssp files are only used in visualization and are imported in the warming code in `climakitaegui` so these are not needed here.

Reply:
Ahh I see the confusion. I forgot to finish up with warming.py. Please review others for now and I will work on this.

Reply:
Sorry should have clarified @elehmer -- AMY breaks without them since it touches `warming.py`

Reply:
Ahh, right. Hopefully will work when I rework warming